rpxdock/app/options.py

```diff
@@ -317,22 +317,27 @@ def default_cli_parser(parent=None, **kw):
    addarg("--sscount_min_sheet_length", default=3, type=int,
           help='Min resis in sheet to count as ss element. default 3')
    addarg("--sscount_min_loop_length", default=1, type=int,
-          help='Min resis in loop to count as ss element. default 1')
-   addarg("--sscount_max_dist", default=8, type=float,
-          help='Min resis in loop to count as ss element. default 1')
+      help='Min resis in loop to count as ss element. default 1')
+   addarg("--sscount_max_dist", type=float, default=8,
+      help='Max distance between residue centroids to be considered in interface. default 8')
    addarg("--sscount_min_element_resis", default=3, type=int,
-          help='Min interface resis in ss_element to include in ss count. default 3')
+      help='Min interface resis in ss_element to include in ss count. default 3')
    addarg("--sscount_sstype", default="EHL", type=str,
-          help='Types of secondary structure to include in count. defaults to all (EHL)')
-   addarg(
-      "--sscount_min_ss_count", default=3, type=int,
-      help='If sscount_confidence set, minimum number of ss elements to pass the filter. default 3'
-   )
-   addarg(
-      "--sscount_strict", action='store_true', default=False,
-      help='Require that both pairs of residues in the interface are in an SS element meeting the set criteria'
-   )
-
+      help='Types of secondary structure to include in count. defaults to all (EHL)')
+   addarg("--sscount_min_ss_count", default=3, type=int,
+      help='If sscount_confidence set, minimum number of ss elements to pass the filter. default 3')
+   addarg("--sscount_strict", action='store_true', default=False,
+      help='Require that both pairs of residues in the interface are in an SS element meeting the set criteria')
+   addarg("--filter_config", default='',
+      help='Path to a yaml file containing the default config file for filters.')
```

Comment:
so if one would not want to parse the default options, one would give a different path right? Can we make it clear in the description? just taking out the default word would work. and may be add the path to the default file afterwards?

Reply:
I could change this to remove the default, that would be fine. Otherwise, if left as-is, the default behavior results in no filters being run which is functionally the same thing. In other words, there is no default filtering.

Reply:
definitely change it, because right now it is not clear. and may be call it example yml here or something of the sort?